webapp/src/components/AssetPage/ItemDetail/ItemDetail.module.css

```
@@ -7,3 +7,69 @@
   flex-direction: column;
   gap: 8px;
 }
+
+.filtertabsContainer {
+  display: flex;
+  padding: 20 0px;
+  border-top-right-radius: 10px;
+  border-top-left-radius: 10px;
+  align-items: center;
+  border-bottom: 1px solid var(--secondary);
+}
+
+.tableContainer{
```

```suggestion
.tableContainer {
```
can we run prettier in this file please?